java/statestore/src/main/java/sleeper/statestore/dynamodb/DynamoDBFileReferenceStore.java

```diff
@@ -122,8 +122,8 @@ public void addFilesWithReferences(List<AllReferencesToAFile> files) throws Stat
     }
 
     private void addFile(AllReferencesToAFile file, Instant updateTime) throws StateStoreException {
-        addFileReferenceCount(file.getFilename(), file.getExternalReferenceCount(), updateTime);
-        for (FileReference reference : file.getInternalReferences()) {
+        addFileReferenceCount(file.getFilename(), 0, updateTime);
```

Comment:
It might be a bit confusing to always pass in 0 to this method. It's only used here, so maybe we should rename it to something like addNewFileReferenceCount and set the reference count to zero in the implementation?